app/extensions/brave/locales/en-US/menu.properties

```diff
@@ -80,6 +80,14 @@ newSessionTab=New Session Tab
 newWindow=New Window
 reopenLastClosedTab=Reopen Last Closed Tab
 print=Print…
+emailPageLink=Email Page Link…
+tweetPageLink=Tweet Page…
+facebookPageLink=Share on Facebook…
```

Comment:
I posted a follow up task marked as good-first-bug for this review comment:
https://github.com/brave/browser-laptop/issues/8604